components/movementsensor/movementsensor.go

```diff
@@ -150,12 +150,13 @@ func NamesFromRobot(r robot.Robot) []string {
 func Readings(ctx context.Context, g MovementSensor, extra map[string]interface{}) (map[string]interface{}, error) {
 	readings := map[string]interface{}{}
 
-	pos, altitide, err := g.Position(ctx, extra)
+	pos, altitude, err := g.Position(ctx, extra)
 	if err != nil && !errors.Is(err, ErrMethodUnimplementedPosition) {
 		return nil, err
 	}
+
 	readings["position"] = pos
-	readings["altitide"] = altitide
+	readings["altitude"] = altitude
```

Comment:
this is a breaking change that should be announced

Reply:
is the process for announcing writing in `#eng-breaking-changes` on slack? and how far before merging should it be announced?

Reply:
announced and added to the ticket for documentation purposes